### DISCUSSION TITLE
OIDC Support

### DISCUSSION THREAD

**Discussion by smolinari** [terrakube-io/terrakube#289] [General]
Hi,

Are there plans to offer OIDC support, like for Keycloak?

Scott

**Comment by alfespa17** [comment]
Hello @smolinari 

Terrakube supports [Dex](https://dexidp.io/) so you can basically connect it with any of the supported Dex Connector like the [OIDC](https://dexidp.io/docs/connectors/oidc/) as long it supports the groups scope. 

You can check the [helm chart repository](https://github.com/AzBuilder/terrakube-helm-chart) for some examples of the Dex Configuration.

There is one example of OIDC with authentication using Amazon Cognito in the following [link](https://github.com/AzBuilder/terrakube-helm-chart/blob/main/examples/CognitoAuthentication-Example1.md), using Keycloak with OIDC should be very similar.

Regards

**Comment by smolinari** [comment]
@alfespa17 - Cool. I'll have a look. And thanks and Merry Christmas!

Scott
